database/migrations/2017_07_30_214254_add_mp3Link_to_tracks.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddMp3LinkToTracks extends Migration
```

Comment:
Since we're starting over, you could just move this to the `CreateTracksTable` migration :)